### PR TITLE
Latex Paper Template

### LINKED ISSUE #2: Large header is repeated on all odd pages, should only be on the first page.
Right now I have it in the odd pages. Works for the abstract because it is only two pages, but needs to be fixed for the paper.

### LINKED ISSUE #3: It is not clear what should and shouldn't be included on the headers and footers after the first page
Should the copyright and issn be on every page?

Should doi be on every page?

### LINKED ISSUE #4: We don't have page numbers
The original odt template had "2 of 3" in the top right on pages after page 1. I think we need fancyheader to get that in the latex doc, but we'd also need to make our header into a fancyheader format, as it is manually overridden in the cls file.

### PR DESCRIPTION
**Pull request by moorepants**
Fixes #2 

Fixes #3 (we will go with this header/footer style)

Fixes #4

### CODE REVIEW

**Comment by moorepants** [comment]
@chrismo-schmidt I have this wort of working. There are some minor issues:

- footer gets centered on some pages
- the margin space above headers on page 2+ are larger
- spacing is not ideal between headers/footers and the body text

I didn't use fancyhdr, but could still try that.

**Comment by moorepants** [comment]
Ok, this seems to now be working.

@chrismo-schmidt Can you review this and see if I should address anything else? Also, maybe you know how to fix #19 ?

After we settle on this template, could you make the new paper templates for odt and docx? Making them match this?

File: paper/paper.tex [hunk 1]
```diff
@@ -0,0 +1,161 @@
+\documentclass{bmd2023p}
+
+\usepackage{lipsum}
```
Comment (moorepants) [inline paper/paper.tex]: remove

**Comment by chris-konrad** [comment]
Hi! Nice work!

This is all the things I have observed:
- Add spaces between numbers and units in the introduction section (e.g. 10 pt).
- Page number is different font size then the rest of the footer.
- Page number is not in the same line as the left part of the footer.
- Example references are not sorted alphabetically. 
- The latex pdf and odt pdf actually don't have the exact same page size. The latex-pdf is 8,28 x 11,70 in and the odt-pdf is 8,27 x 11,69 in. I could adapt the odt page size but it feels wrong to change the A4 preset. 

In all other aspects, I adjusted the odt template to look like the latex template. The page number and the page size are the only things that I would suggest to change in latex. 

About #19: There is a wrong whitespace in the superscript:
Odt:
image 
Latex:
image

Should be an easy fix by removing "~" here:
https://github.com/moorepants/bmd-templates/blob/9dedf1b600b22d3f8b80b34e6cbd01526bd3fa6e/paper/bmd2023p.cls#LL131C24-L131C25

I'll create a word template as soon as we are finished with latex/odt. 

Here a side-by-side of latex and my current odt template. Different line width might be a rendering issue. 
image

**Comment by moorepants** [comment]
> Add spaces between numbers and units in the introduction section (e.g. 10 pt).

The tilde (~) in latex ensures that no line breaks will occur between, for example, "10" and "pt". The tilde must also tighten the spacing between the attached characters. If you want more space between tilded characters we would have to edit the spacing rules for the tilde, but I'd rather just use LaTeX's default.

**Comment by moorepants** [comment]
I see where I have `10pt` in some places, it needs a tilde. That must be what you were talking about.

**Comment by moorepants** [comment]
> About #19: There is a wrong whitespace in the superscript:

I'm not quite sure how to fix this. This is the line but the tilde doesn't seem to make a tight spacing:

```
▷⋅⋅⋅⋅⋅⋅⋅\listadd{\authorlist}{#1~\textsuperscript{#2}}}
```

**Comment by moorepants** [comment]
> Page number is different font size then the rest of the footer.
> Page number is not in the same line as the left part of the footer.

Fixed with:

fb38631ab07a7429bd09754b754232fe99fb6972

**Comment by moorepants** [comment]
> The latex pdf and odt pdf actually don't have the exact same page size. The latex-pdf is 8,28 x 11,70 in and the odt-pdf is 8,27 x 11,69 in. I could adapt the odt page size but it feels wrong to change the A4 preset.

Fixed with fd63207

**Comment by moorepants** [comment]
I've fixed everything but the superscript spacing on the author names. I'm not actually sure how to do that without removing the tildes (with may be needed for long author lists).

**Pull request merged**